src/core/package.ml

```diff
@@ -75,6 +75,15 @@ let read : file_path -> config_data = fun fname ->
     [Sys_error] is raised. Note that [fname] is first normalized with a call
     to [Filename.realpath]. *)
 let find_config : file_path -> file_path option = fun fname ->
+  let rec normalize fname =
```

Comment:
You already defined this function in files.ml. This should be factorized by adding a function in filename.ml.

Reply:
done